plugins/modules/inventory_workflow_manager.py

```diff
@@ -863,7 +868,7 @@ def get_device_ips_from_config_priority(self):
             If none of the information is available, an empty list is returned.
         """
         # Retrieve device IPs from the configuration
-        device_ips = self.config[0].get("ip_address_list")
+        device_ips = self.want.get("device_params").get("ipAddress")
```

Comment:
Why we are taking 'ipAddress' in camel case from the playbook as we are strictly bound to use snake_case for variable and method name.
Also make sure to check first you receive device_params from playbook or not and then fetch the ipAddress as device_params is None then module will come into exception resulting in failure without proper message.

Reply:
its not from the playbook. we discussed in the call